extensions/youtube-music-desktop-app-controller/package.json

```
@@ -0,0 +1,145 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "youtube-music-desktop-app-controller",
+  "title": "Youtube Music Desktop App Controller",
```

```suggestion
  "title": "Youtube Music Desktop",
```
What about this name/title

---

Updated in https://github.com/raycast/extensions/pull/14379/commits/91fad14bf6a729551bf62f7b6826997bdbf8c9ee